spec/controllers/people_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+# spec/controllers/people_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe PeopleController, type: :controller do
+  describe "GET #index" do
+    subject { get :index }
+
+    it "responds with 200" do
+      expect(subject.status).to eq(200)
```

Comment:
No need to change this, but I want to mention that The Response object here responds to a number of semantic 'status' methods like `:successful` allowing you to write something like `expect(subject).to be_successful` 

http://www.rubydoc.info/gems/rack/Rack/Response (included in ActionDispatch::Response)